src/block-management/block-manager.js

```diff
@@ -72,16 +72,11 @@ export default class BlockManager extends React.Component<PropsType, StateType>
 				this.props.deleteBlockAction( clientId );
 				break;
 			case ToolbarButton.PLUS:
-				// TODO: direct access insertion: it would be nice to pass the dataSourceBlockIndex here,
-				// so in this way we know the new block should be inserted right after this one
-				// instead of being appended to the end.
-				// this.props.createBlockAction( uid, dataSourceBlockIndex );
-
 				// TODO: block type picker here instead of hardcoding a core/code block
 				const newBlock = createBlock( 'core/paragraph', { content: 'new test text for a core/paragraph block' } );
 				const newBlockWithFocusedState = { ...newBlock, focused: false };
```

Comment:
I know it's not part of the PR's diff but, wdyt about doing a small optimization here by just appending to the object a new field instead of creating a full new object?

We can just do:
```
newBlock.focused = false;
```
and use the `newBlock` instead of construct the `newBlockWithFocusedState`.

Reply:
good call! decided to address that in 39f6500 in #112 given this part of the code gets moved around in there 👍